### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by thomasheartman**

### CODE REVIEW

**Comment by thomasheartman** [comment]
[Update README on GitHub](https://trello.com/c/eKuEDgUo/1019-update-readme-on-github)

**Review by chriswk — APPROVED** [review]
Looks good!

File: README.md [hunk 1]
```diff
@@ -1,18 +1,98 @@
...
+If you don't want to run Unleash locally, we also provide easy deployment setups for Heroku and Digital Ocean:
+
+[![Deploy to Heroku](./.github/deploy-heroku-20.png)](https://www.heroku.com/deploy/?template=https://github.com/Unleash/unleash) [![Deploy to DigitalOcean](./.github/deploy-digital.svg)](https://cloud.digitalocean.com/apps/new?repo=https://github.com/Unleash/unleash/tree/main&refcode=0e1d75187044)
+
+## Online demo
+
+Try out [the Unleash online demo](https://www.getunleash.io/interactive-demo).
 
-Unleash comes with [official client SDKs](https://docs.getunleash.io/sdks) for all the popular languages.
+[Get image from Nicolae here]
```
Comment (chriswk) [inline README.md:76]: TODO: This one needs fixing before we merge this

**Review by chriswk — CHANGES_REQUESTED** [review]
Putting a blocker until we no longer have the reminder to ask Nicolae about image.

**Comment by thomasheartman** [comment]
@chriswk For sure! I'd just forgotten to mark the PR as draft. But the reminder is gone now and the PR is ready 🚂

**Review by chriswk — APPROVED** [review]
LGTM :)

**Pull request merged**